src/components/logs/SpinnerIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import { AlertColor, CircularProgress } from '@mui/material';
+import { lineNumberColor } from './Line';
+
+interface Props {
+    stopped: boolean;
+    severity?: AlertColor;
+}
+
+function SpinnerIcon({ severity, stopped }: Props) {
```

Comment:
This was initially broken out to make some other work easier. I left it broken out as it does not hurt anything and makes the code more easily updated in the future.